src/components/Input/Input.module.css

```diff
@@ -7,7 +7,7 @@
 /**
  * Default text input styles 
  */
-:where(.input) {
+.input {
```

Comment:
Want this more specific than preflight. This is a subcomponent anyways so I don't expect overrides outside of EDS (if they're even necessary)